packages/buendia-site-test-integration/data/usr/share/buendia/site/40-site

```diff
@@ -17,3 +17,6 @@ MONITORING_LIMITS='
 '
 
 UPDATE_AUTOUPDATE=1
+
+# Enable destructive testing for this site by default.
+UTIL_RUN_DESTRUCTIVE_TESTS=1
```

Comment:
The convention for site settings is that they start with the package names of the things they control, so since the testing stuff is in `buendia-utils`, the `UTIL_RUN_DESTRUCTIVE_TESTS` setting should start with `UTILS_`.  I'm guessing maybe you meant this to be the case for the other `UTIL_...` variables also?

Reply:
Yes, that was an error. Fixed